llvm/lib/Target/ARM/AsmParser/ARMAsmParser.cpp

```diff
@@ -6340,7 +6340,7 @@ StringRef ARMAsmParser::splitMnemonic(StringRef Mnemonic,
     unsigned CC = ARMCondCodeFromString(Mnemonic.substr(Mnemonic.size()-2));
     if (CC != ~0U) {
       Mnemonic = Mnemonic.slice(0, Mnemonic.size() - 2);
-      PredicationCode = CC;
+      PredicationCode = (ARMCC::CondCodes)CC;
```

Comment:
https://llvm.org/docs/CodingStandards.html#prefer-c-style-casts

Reply:
The anchor link really changes the meaning of that link (:
I'll get a quick fix in